### PR TITLE
Address NUnit repeat failure

### LINKED ISSUE #396: Repeat tests fail
The repeat functionality for NUnit (https://github.com/nunit/docs/wiki/Repeat-Attribute) breaks when doing anything other that base tests.

The issue is that in our test teardown we create a new base test object for logging purposes but never cleanup after it:
https://github.com/Magenic/MAQS/blob/4165e61ae80cb9d05d3b31fac98d3817e371ee54/Framework/BaseTest/BaseTest.cs#L296
The upshot is that the second time the test runs it will pickup the logging base test object.

This was found in #390

### PR DESCRIPTION
**Pull request by TroyWalshProf**
Closes #396

### CODE REVIEW

File: Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs [hunk 1]
```diff
@@ -8,8 +8,10 @@
 using Magenic.Maqs.BaseWebServiceTest;
 using Magenic.Maqs.Utilities.Helper;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
```
Comment (FermJacob) [inline Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs:11]: Do we want to separate this out into a specific NUnit class?

ex. https://github.com/Magenic/MAQS/blob/20e7051fa85d4afdc2924051de14569a491f0d75/Framework/WebServiceUnitTests/WebServiceNUnit.cs

File: Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs [hunk 2]
```diff
@@ -102,5 +104,18 @@ public void NotIntialized()
             WebServiceDriverManager driverDriver = this.ManagerStore[typeof(WebServiceDriverManager).FullName] as WebServiceDriverManager;
             Assert.IsFalse(driverDriver.IsDriverIntialized(), "The driver should not be initialized until it gets used");
         }
+
+        /// <summary>
+        /// Check that tests can run as repeat
+        /// </summary>
+        [Test]
```
Comment (FermJacob) [inline Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs:111]: Do we want to add a TearDown attribute here or a specific test calling the TearDown to check the finally statement?  Not sure if its possible to hit that area

File: Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs (hunk 1 above)
Comment (TroyWalshProf) [reply Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs]: You convinced me

File: Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs (hunk 2 above)
Comment (TroyWalshProf) [reply Framework/FrameworkUnitTests/WebServiceDriverManagerTests.cs]: I don't think we have that level of control

**Pull request merged**